### DISCUSSION TITLE
欢迎来到 ONES Group 讨论区

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#1] [日常讨论]
## 👋 欢迎!
  我们使用 Discussions 的功能来连接社区的所有成员. 我们希望：
  * 积极提问你想知道的关于开源治理与社区运营的事宜.
  * 分析你的想法.
  * 和大家一起来协作.
  * 开放协作，一期建设这个社区 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tedliu1** [comment]
Hi Everyone

**Comment by wangleo61** [comment]
支持王老师
